packages/open-truss/src/configuration/engine-v1/RenderConfig.tsx

```diff
@@ -48,7 +52,14 @@ export function RenderConfig({
     configPath,
     COMPONENTS,
   ) as FrameWrapper
+
+  const signals = createSignals(config.signals)
```

Comment:
Updated the code to create signals based on what signals are declared in the config